sponsors.html

```diff
@@ -15,4 +15,16 @@ <h3 class="clemson-orange-text">By sponsoring, you gain ...</h3>
   </ul>
   <h3 class="clemson-orange-text">How to sponsor</h3>
   <p class="flow-text">For more information on ways to become involved with <strong>CUCTF 2020</strong>, please download our <a href="/public/files/CUCTF_Sponsorship_Packet.pdf" download="">Sponsor Packet</a> or contact us at <a href="mailto:sponsor@cuctf.io">sponsor@cuctf.io</a>.</p>
+  <br>
+  <h4 class="clemson-orange-text">CUCTF 2020 Contingency Plan for Sponsors</h4>
+  <div class="card black">
+      <div class="card-content">
+        <p> In the case that CUCTF 2020 switches to an online infrastructure, all physical event sponsors <i>(Silver, Gold, and Diamond)</i> will: </p>
+        <ul class="browser-default">
+          <li>Have the opportunity to connect with students through Zoom meeting lobbies</li>
+          <li>Maintain sponsorship status into the CUCTF 2021 Event</li>
```

Comment:
We say below all sponsors will be carried into CUCTF 2021 (which makes sense) but the bullet point is limited here to just silver, gold, and diamond. Maybe either clarify here that this also applies to all sponsors or clarify the below to explicitly say all sponsor tiers will carry over.